docs/source/app-dev/bindings-java/codegen.rst

```diff
@@ -14,7 +14,7 @@ When writing applications for the ledger in Java, you want to work with a repres
 Download
 ========
 
-You can download the `latest version <https://bintray.com/api/v1/content/digitalassetsdk/DigitalAssetSDK/com/daml/java/codegen/$latest/codegen-$latest.jar?bt_package=sdk-components>`__  of the Java codegen. Make sure that the following versions are aligned:
+You can download the `latest version <https://search.maven.org/artifact/com.daml.java/codegen>`__  of the Java codegen. Make sure that the following versions are aligned:
```

Comment:
Based on the changes, shouldn't it be:

```suggestion
You can download the `latest version <https://search.maven.org/artifact/com.daml.bindings.java/codegen>`__  of the Java codegen. Make sure that the following versions are aligned:
```

Reply:
Shouldn't we direct users to use `daml codegen java` instead of installing a codegen jar separately?

Reply:
Thank you 🙏 
Yes, this needs to be brought into today's codegen world via the daml assistant.